.gitignore

```diff
@@ -17,4 +17,9 @@ override.tf.json
 !.yarn/plugins
 !.yarn/releases
 !.yarn/sdks
-!.yarn/versions
\ No newline at end of file
+!.yarn/versions
+
+dist
+.env
+node_modules
```

Comment:
Do we need `node_modules`?

Reply:
yes, because it will be generated by our dev tools (such as vitest...)